tests/modules/collaborations/resources/test_collaboration_patch.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+import tests.modules.collaborations.resources.utils as collab_utils
+from tests import utils
+
+
+def test_patch_collaboration(
+    flask_app_client, researcher_1, readonly_user, researcher_2, db
+):
+    from app.modules.collaborations.models import Collaboration
+
+    collab = None
+    try:
+        data = {'user_guid': str(researcher_2.guid)}
+        collab_utils.create_collaboration(flask_app_client, researcher_1, data)
+        collabs = Collaboration.query.all()
+        collab = collabs[0]
```

Comment:
This won't work when testing with a production DB or something like the soon to be pseudo-migrated zebra database.

Reply:
The same applies to a whole ton of tests. I'm afraid that will be a big task when we do that.